bin/plugin/commands/changelog.js

```diff
@@ -757,6 +775,94 @@ function sortFeatureGroups( featureGroups ) {
 	);
 }
 
+/**
+ *
+ * @param {IssuesListForRepoResponseItem[]} pullRequests List of pull requests.
+ *
+ * @return {IssuesListForRepoResponseItem[]} pullRequests List of first time contributor PRs.
+ */
+function getFirstTimeContributorPRs( pullRequests ) {
+	return pullRequests.filter( ( pr ) => {
+		if ( pr.user.login === 'dependabot[bot]' ) {
```

Comment:
It should be possible to make this work for all bots using the `type` field that github provides on the `user` object (`pr.user.type === 'Bot'`).

See https://github.com/WordPress/gutenberg/pull/38393 for reference.

Reply:
Ooooo nice. Thanks Dan 🙇‍♂️

Reply:
✅